src/modules/staffAssignment/components/MyClients.tsx

```diff
@@ -37,31 +39,65 @@ const MY_CLIENTS_COLUMNS: ColumnDef<StaffAssignmentWithClientsFragment>[] = [
   {
     header: 'Head of Household',
     render: (assignment) => clientBriefName(memoizedHoh(assignment).client),
+    key: 'headOfHousehold',
   },
   {
     header: 'Members',
     render: (assignment) => assignment.household.householdClients.length,
+    key: 'members',
   },
   {
     header: 'Project',
     render: (assignment) =>
       memoizedHoh(assignment).enrollment.project.projectName,
+    key: 'project',
   },
   {
-    header: 'Enrollment Period',
+    ...WITH_ENROLLMENT_COLUMNS.entryDate,
     render: (assignment) => {
-      const hoh = memoizedHoh(assignment);
-      return parseAndFormatDateRange(
-        hoh.enrollment.entryDate,
-        hoh.enrollment.exitDate
+      return renderCellContents(
+        memoizedHoh(assignment),
+        WITH_ENROLLMENT_COLUMNS.entryDate.render
       );
     },
+    tableCellProps: undefined, // typescript
```

Comment:
this seems odd, is there an underlying change we can make to the ColumnDef type, or some additional type hints here?